src/VirtoCommerce.Platform.Core/ChangeLog/OperationLog.cs

```diff
@@ -36,5 +37,21 @@ public virtual object Clone()
         }
 
         #endregion
+
+        /// <summary>
+        /// Find the type name for operation log. It's a direct successor of AuditableEntity by default.
+        /// </summary>
+        /// <returns>Name of found type</returns>
+        protected virtual string GetTypeNameForLogging<T>(T entry) where T : IEntity
+        {
+            if (entry is AuditableEntity)
+            {
+                return entry.GetType().GetTypeInheritanceChainTo(typeof(AuditableEntity)).Last().Name;
+            }
+            else
+            {
+                return entry.GetType().Name;
+            }
```

Comment:
```suggestion
            else if (entry is Entity)
            {
                return entry.GetType().GetTypeInheritanceChainTo(typeof(Entity)).Last().Name;
            }
            else
            {
                return entry.GetType().Name;
            }
```
There are many types that do not derive the `AuditableEntity` but still can be extended - this approach will work consistently with these types.

Reply:
Interesting, that I could not find any descendant of Entity, that would be logged (All were AuditableEntity)